cmd/tnf/generate/catalog/catalog.go

```diff
@@ -173,6 +184,54 @@ func outputTestCases() {
 	fmt.Println()
 }
 
+func outputJS() {
```

Comment:
Proposed simplification:
Instead of performing the bulk of the logic here, use the AddCatalogEntry to build a classification map:
```
Classification[aID.Id] = categoryclassification
```
where
```
var Classification = map[string]map[string]string{}
```
in the outputJS function, simply marshall into Json:
```
func outputJS() {
	out, err := json.MarshalIndent(identifiers.Classification, "", "  ")
	if err != nil {
		logrus.Errorf("could not Marshall classification, err=%s", err)
		return
	}
	fmt.Printf("window.classification= JSON.Parse ( %s )", out)
}
```
In the HTML file, the json file should be parsed into a variable.